lib/elixir/lib/task.ex

```diff
@@ -735,8 +778,8 @@ defmodule Task do
   Returns `{:ok, reply}` if the reply is received while shutting down the task,
   `{:exit, reason}` if the task died, otherwise `nil`.
 
-  The second argument is either a timeout or `:brutal_kill`. In case
-  of a `timeout`, a `:shutdown` exit signal is sent to the task process
+  The second argument is either a timeout value in milliseconds, :infinity or `:brutal_kill`.
```

Comment:
I think it makes no sense to accept `:infinity` as a timeout, please correct me if I'm overseeing something.
This is already checked in the guards but doc and spec needs to be updated